### DISCUSSION TITLE
cannot download

### DISCUSSION THREAD

**Discussion by planeman1** [remix-run/remix#9244] [Q&A]
im stuck at the command input stage, can some one help by giving an detail instruction from downloading of github file to ide set up please
![image](https://github.com/remix-run/remix/assets/120025433/d5e85961-78bc-43a0-8b5b-002f7a901228)
last cmb error

**Comment by sergiodxa** [comment]
The command to create a new Remix app is this
```sh
npx create-remix@latest
```
There's no need to install a global package, but if you want to, the package is `create-remix`.
